https://github.com/torvalds/linux/commit/c8e2d59bf4ddebf4ceafd4e902e572a045e6ae10

RDMA/mthca: Delete useless module.h include

There is no need in include of module.h in the following file.

Link: https://lore.kernel.org/r/ab856f40804d67905a655bc85e480d96ff66e46e.1642960861.git.leonro@nvidia.com
Signed-off-by: Leon Romanovsky <leonro@nvidia.com>
Signed-off-by: Jason Gunthorpe <jgg@nvidia.com>

diff -Naurp ib_mthca-1.0.20080404.orig/mthca_profile.c ib_mthca-1.0.20080404/mthca_profile.c
--- ib_mthca-1.0.20080404.orig/mthca_profile.c	2024-04-10 09:08:06.000000000 -0400
+++ ib_mthca-1.0.20080404/mthca_profile.c	2024-09-09 16:49:56.395466642 -0400
@@ -31,8 +31,6 @@
  * SOFTWARE.
  */
 
-#include <linux/module.h>
-#include <linux/moduleparam.h>
 #include <linux/string.h>
 #include <linux/slab.h>
 
